lib/workers/repository/updates/generate.ts

```diff
@@ -84,9 +84,9 @@ export function generateBranchConfig(
     toValues.add(upg.newValue!);
     // prettify newVersion and newMajor for printing
     if (upg.newVersion) {
-      upg.prettyNewVersion = upg.newVersion.startsWith('v')
-        ? upg.newVersion
-        : `v${upg.newVersion}`;
+      upg.prettyNewVersion = semver.valid(upg.newVersion)
```

Comment:
not sure if we should only do this for semver compliant values. 🤔

Reply:
Me too, but for now, all unit test cases are related to semantic versioning

Reply:
Can we use the selected versioning for the upgrade plus maybe a check that it starts with a number?

Reply:
Added function `prettifyVersion`